internal/pkg/cli/init.go

```diff
@@ -540,6 +564,50 @@ func (o *initOpts) askShouldDeploy() error {
 	return nil
 }
```

Comment:
Should probably error out if both flags are set, just to make the behavior obvious, because in this case setting both `--no-deploy` and `--deploy` would set `o.ShouldDeploy` to true.